content/docs/get-started/azure/modify-program.md

```diff
@@ -12,161 +12,235 @@ menu:
+    ResourceGroupName: resourceGroup.Name,
+    IndexDocument:     pulumi.String("index.html"),
+})
+if err != nil {
+    return err
+}
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+```csharp
+// Enable static website support
+var staticWebsite = new StorageAccountStaticWebsite("staticWebsite", new StorageAccountStaticWebsiteArgs
+{
+    AccountName = storageAccount.Name,
+    ResourceGroupName = resourceGroup.Name,
+    IndexDocument = "index.html",
 });
+```
+
+{{% /choosable %}}
+
+The static website resource leverages the storage account and resource group names defined previously in your program.
 
-// Export the connection string for the storage account
-export const connectionString = account.primaryConnectionString;
+Now use all of these cloud resources, and a local `FileAsset` resource, to upload `index.html` into your storage container.
+
+{{% choosable language typescript %}}
+
+```typescript
+// Upload the file
+["index.html"].map(name =>
+    new storage.Blob(name, {
+        blobName: name,
+        resourceGroupName: resourceGroup.name,
+        accountName: account.name,
+        containerName: staticWebsite.containerName,
+        type: storage.BlobType.Block,
+        source: new pulumi.asset.FileAsset(`./site/${name}`),
+        contentType: "text/html",
+    }),
+);
 ```
 
 {{% /choosable %}}
+
 {{% choosable language python %}}
 
 ```python
-import pulumi
-from pulumi_azure import core, storage
+# Upload the file
+index_html = storage.Blob("index_html",
+    blob_name="index.html",
+    resource_group_name=resource_group.name,
+    account_name=account.name,
+    container_name=static_website.container_name,
+    type=storage.BlobType.BLOCK,
+    source=pulumi.FileAsset("./site/index.html"),
+    content_type="text/html")
+```
 
-# Create an Azure Resource Group
-resource_group = core.ResourceGroup("resource_group")
+{{% /choosable %}}
+{{% choosable language go %}}
 
-# Create an Azure resource (Storage Account)
-account = storage.Account("storage",
-    resource_group_name=resource_group.name,
-    account_tier='Standard',
-    account_replication_type='LRS',
-    tags={"Environment": "Dev"})
+```go
+// Upload the file
+_, err = storage.NewBlob(ctx, "index_html", &storage.BlobArgs{
+    BlobName:          pulumi.String("index.html"),
+    ResourceGroupName: resourceGroup.Name,
+    AccountName:       account.Name,
+    ContainerName:     staticWebsite.ContainerName,
+    Type:              storage.BlobTypeBlock,
+    Source:            pulumi.NewFileAsset("./site/index.html"),
+    ContentType:       pulumi.String("text/html"),
+})
+if err != nil {
+    return err
+}
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
 
-# Export the connection string for the storage account
-pulumi.export('connection_string', account.primary_connection_string)
+```csharp
+// Upload the file
+var index_html = new Blob("index_html", new BlobArgs
+{
+    BlobName = "index.html",
```

Comment:
You could choose to pass `new Blob("index.html", ...)` and then omit the `BlobName` assignment - Azure-Native would automatically copy that name as-is.